client/src/pages/positions/Show.js

```diff
@@ -199,31 +199,12 @@ const PositionShow = ({ pageDispatchers }) => {
                   label={Settings.fields.position.name}
                 />
 
-                <CodeFieldWithLabel
```

Comment:
It makes sense to keep the same order of fields on the position Form.

Reply:
There are sections that are shown or not the same in some form and show page or preview structures, or even if they are the same, I have adjusted them to be more comfortable and more useful to the eye.